### PR TITLE
fix: load env variables

### PR DESCRIPTION
**Pull request by josethz00**
This PR fixes the environment variables load into the application

### CODE REVIEW

File: configs/environment.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import os, dotenv
+
+def load_environment():
+    dotenv.load_dotenv()
```
Comment (NicolasLopes7) [inline configs/environment.py:4]: why is this method not part of the class?

Comment (josethz00) [reply configs/environment.py:4]: makes sense, thank you fella

File: main.py [hunk 2]
```diff
@@ -12,6 +13,9 @@ def getDb():
 
 app = FastAPI()
 
+load_environment()
+print(Environment.DB_USER)
```
Comment (NicolasLopes7) [inline main.py:17]:
```suggestion
```

**Review by NicolasLopes7 — APPROVED** [review]
left some non blocking comments. looks good at all

**Comment by josethz00** [comment]
> left some non blocking comments. looks good at all

sure, I'll do

**Comment by anddreluis2** [comment]
LGTM

**Review by lxcvz — APPROVED** [review]

**Pull request merged**